src/components/ResponsiveList/ResponsiveList.jsx

```diff
@@ -135,4 +140,6 @@ ResponsiveList.defaultProps = {
   resizeDebounceTime: 0
 };
 
+ResponsiveList.defaultTestId = ComponentDefaultTestId.RESPONSIVE_LIST;
```

Comment:
why is this needed? I saw it also in other components but I don't see why it's necessary

Reply:
Basically it serves two points:
1. Get default data-testid of the component without importing this big file with all the constants
2. See this defaul data-testid in the component Storybook props table

Reply:
Why not simply export the default test ids file instead of re-exporting per component.
Also thinking about our cypress tests - I don’t want users to import jsx files - only raw js files

Reply:
So currently we are exporting this file with constants in `interactionTests.js`
This defaultTestId is supposed to be a solution when user don't want to bring in whole `interactionTests.js`, just in order to get, let's say Clickable default data-testid - he'd be able just to get from `Clickable.defaultTestId`

Reply:
I think it’s better to have one way to use it instead of multiple ones and the single file is better in terms of cross-platform testing

Reply:
Removed in Vibe 2.0 https://github.com/mondaycom/monday-ui-react-core/pull/1254